src/SFML/Graphics/RenderTextureImplFBO.cpp

```diff
@@ -594,13 +594,13 @@ void RenderTextureImplFBO::updateTexture(unsigned int)
 
         Lock lock(mutex);
 
-        std::map<Uint64, unsigned int>::iterator iter = m_frameBuffers.find(contextId);
-        std::map<Uint64, unsigned int>::iterator multisampleIter = m_multisampleFrameBuffers.find(contextId);
+        auto it = m_frameBuffers.find(contextId);
+        auto multisampleIter = m_multisampleFrameBuffers.find(contextId);
```

Comment:
Maybe name the two `frameBufferIt` and `multisampleIt`?

Reply:
Will fix